pkg/serverless/invocationlifecycle/rand.go

```diff
@@ -0,0 +1,56 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
This file handles the generation of random trace and span IDs. I copied it over from dd-trace-go to match the behavior of the Lambda Library.